components/SignInModal.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import Container from './Container';
+import { Box, Flex } from './Grid';
+import Link from './Link';
+import StyledButton from './StyledButton';
+import StyledCard from './StyledCard';
+import StyledInput from './StyledInput';
+import StyledLink from './StyledLink';
+import { H5, Span } from './Text';
+
+/**
+ * TODO: Remove once we decide on the modal that appear in the CommentForm
+ */
+export default class SignInModal extends React.Component {
```

Comment:
In `CommentForm.js` we show the `SignIn` component as a card with a `ContainerOverlay`; 

https://github.com/opencollective/opencollective-frontend/blob/e2e1b4524b2b8c8300e75a81bcf358a7689d4a1a/components/conversations/CommentForm.js#L123-L129

I couldn't think of a way to adapt the new signin form in order to do the same thing. Maybe we need some design guidance on this but until then, I am proposing this method where we have this `SignInModal` component which basically shows the sign in component just like the way we did before (only in the `CommentForm`).

![image](https://user-images.githubusercontent.com/12435965/173972600-bd64f736-071f-4335-91e9-1bffaa396ae9.png)